atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -215,14 +201,80 @@ def _send_mail(self, receivers_list, subject, body):
 
         s = None
         try:
-            s = smtplib.SMTP(self.smtp_uri)
+            s = smtplib.SMTP(self.smtp_host)
             s.sendmail(self.from_address, receivers_list, msg.as_string())
         except (socket.gaierror, smtplib.SMTPException) as e:
             raise PluginFailedException('Error communicating with SMTP server: %s' % str(e))
```

Comment:
The above two lines (exception handling for `s.sendmail()`) don't get covered by unit testing.